markdown/about.md

```diff
@@ -71,6 +70,11 @@ The current members of the maintainer's team in 2023 are: <!-- TODO: check if ye
 
 We thank all the institutions and companies of the maintainers team for their support of nf-core.
 
+## Maintainers alumni {#maintainers-alumni}
+
+- 2022-2023
+  - [@ggabernet](https://github.com/ggabernet): Gisela Gabernet
+
```

Comment:
```suggestion
```

Reply:
I would prefer to just be removed of that team and not be listed as alumni if that's ok

Reply:
Yeah sure!